src/app/parcels/ParcelsPage.tsx

```diff
@@ -458,35 +459,33 @@ const ParcelsPage: React.FC<{}> = () => {
 
     useEffect(() => {
         if (!areFiltersLoadingForFirstTime) {
-            const allFilters = [...primaryFilters, ...additionalFilters];
-            const initialRequestParams: RequestParams<ParcelsTableRow> = {
-                allFilters: { ...allFilters },
-                sortState: { ...sortState },
-                startPoint: startPoint,
-                endPoint: endPoint,
-            };
-            (async () => {
-                setIsLoading(true);
-                const filteredParcelCount = await getParcelsCount(supabase, allFilters);
-                const fetchedData = await getParcelsData(
+            const allFilters = [...primaryFilters.slice(), ...additionalFilters.slice()];
```

Comment:
Is this another weird array behaviour that required you to do this?
```suggestion
            const allFilters = [...primaryFilters, ...additionalFilters];
```
or even
```suggestion
            const allFilters = primaryFilters.concat(additionalFilters);
```

Reply:
Nope, this was leftover from when I was trying to fix the weird array behaviour, but we didn't actually need this, it had no effect. have removed